src/subdomains/generic/user/models/ip/ip.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { IEntity } from 'src/shared/models/entity';
+import { Entity, Column } from 'typeorm';
+
+@Entity()
+export class Ip extends IEntity {
```

Comment:
Maybe rename it to SignInLog or similar?

Reply:
renamed it to IpLog